tools/test/h5dump/CMakeVFDTests.cmake

```diff
@@ -24,14 +24,33 @@ set (HDF5_VFD_H5DUMP_FILES
   packedbits
 )
 
+set (HDF5_SF_VFD_H5DUMP_FILES
+  test_subfiling_stripe_sizes.h5
+)
+
 foreach (vfdtest ${VFD_LIST})
+  if (vfdtest STREQUAL "subfiling")
+    foreach (h5_tfile ${HDF5_SF_VFD_H5DUMP_FILES})       
+      file(COPY "${PROJECT_SOURCE_DIR}/testfiles/${h5_tfile}" DESTINATION "${PROJECT_BINARY_DIR}/${vfdtest}")
+      execute_process(
```

Comment:
Why is this doing a process that could be done by a CMake command? We should not be using platform specific commands by choice. We should not assume that something is only ever going to be on one platform.

Reply:
Why use the inode in the first place - the files are unique without the number. I don't see the advantage of the complexity.

Reply:
The inode is used by the Subfiling VFD to generate unique filenames, so the CMake code would need to be able to generate the names of the other resulting files when creating files with the Subfiling VFD during testing.

Reply:
But isn't that in the original name? Seems like there needs to be more documentation on this test, because it isn't obvious.

Reply:
The problem is we can't add generated files here easily because the inode value for the file could change, meaning that when the Subfiling VFD tries to open `test_subfiling_stripe_sizes.h5`, it will also look for 
`test_subfiling_stripe_sizes.h5.subfile_XXXXXXXX_1_of_1` and `test_subfiling_stripe_sizes.h5.subfile_XXXXXXXX.config`, but we wouldn't have a way of predicting what the inode value for `XXXXXXXX` would be and so we have to grab the inode value. On the other hand, we probably don't necessarily need a generated file at all to test Subfiling with h5dump if we just create the file on the fly.